composer.json

```diff
@@ -72,7 +71,7 @@
             "bin-links": false
         },
         "branch-alias": {
-            "dev-master": "1.0-dev"
+            "dev-master": "0.14-dev"
         }
```

Comment:
This was the blocker actually. As the `humbug/box` package depends on 0.14, this version cannot be higher. It would be using different version than allowed.